async-dog/index.html

```diff
@@ -19,7 +19,18 @@ <h2>대신 귀여운 강아지를 드립니다.</h2>
 
     <script>
       const url = "https://dog.ceo/api/breeds/image/random";
-      // TODO: url에서 강아지 사진 링크를 가져와 본문의 img 소스를 대체하기
+
+      const onChangeImage = event => {
+        event.preventDefault();
+        fetch(url)
+                .then(data => data.json())
+                .then(data => {
+                  document.getElementById("dog-image").src = data.message;
+                })
```

Comment:
Fetch API를 사용할 때는 우선 `response`를 받고, 파싱 후 `data`라는 변수로 처리하는 것이 더 명확합니다.
그리고 `message` 변수만 사용하는 이 경우 [구조 분해 할당](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)을 사용할 수 있습니다.

```suggestion
        fetch(url)
                .then(response => response.json())
                .then(({message}) => {
                  document.getElementById("dog-image").src = message;
                })
```

Reply:
변수를 명확히 수정하고 구조 분해 할당을 사용해보았습니다!